assets/Script/game/fallingBlock.js

```diff
@@ -13,9 +13,9 @@ cc.Class({
   extends: cc.Component,
 
   properties: {
-    speed: 50,
-    movingHorizontal: true,
-    movingVertical: true
+    speed: 200,
+    fadeOutTime: 0.4,
+    blockType: 'blue'
```

Comment:
Until we settle on how to do the 'block matching', might be worth leaving this out.